site/src/routes/metrics.js

```diff
@@ -0,0 +1,154 @@
+import { stores } from '../constants.js'
+import { get as getDeals } from '../models/deals.js'
+
+const MAX_AGE_SECS = 60 * 10 // max age of a metrics response in seconds
```

Comment:
we can do at least 1 hour here right ?

Reply:
Yep, depends what resolution we want to see...